.teamcity/builds/kotlinlang/buidTypes/FetchBlogNews.kt

```diff
@@ -1,19 +1,77 @@
 package builds.kotlinlang.buidTypes
 
+import jetbrains.buildServer.configs.kotlin.BuildSteps
 import jetbrains.buildServer.configs.kotlin.BuildType
+import jetbrains.buildServer.configs.kotlin.CheckoutMode
+import jetbrains.buildServer.configs.kotlin.buildFeatures.sshAgent
+import jetbrains.buildServer.configs.kotlin.buildSteps.ScriptBuildStep
 import jetbrains.buildServer.configs.kotlin.buildSteps.script
+import jetbrains.buildServer.configs.kotlin.triggers.schedule
+
+private fun BuildSteps.nodejs(block: ScriptBuildStep.() -> Unit) = step(
+    ScriptBuildStep {
+        dockerImage = "node:20"
+        dockerPull = true
+    }.apply(block),
+)
+
+private const val DATA_PATH = "latest-news"
+private const val SCRIPT_PATH = "scripts/latest-news"
 
 object FetchBlogNews : BuildType({
     name = "Fetch Blog News"
 
+    artifactRules = """
+        $DATA_PATH => latest-news.zip
+    """.trimIndent()
+
+    triggers {
+        schedule {
+            schedulingPolicy = cron {
+                hours = "*/1"
```

Comment:
Let's run it not so often, no more than once a day. I suggest running it in the middle of the day (12CET?)

Reply:
https://github.com/JetBrains/kotlin-web-site/pull/4002/files#diff-c4663856d871aa2e35d00ca8e372e2f34916751617577132f3a43476f39750ecR30-R31